collections/_topics/emisie.md

```diff
@@ -4,30 +4,157 @@ title:         "Emisie skleníkových plynov a emisné opatrenia"
+
+    Klimatická zmena závisí na celkovom množstve skleníkových plynov v atmosfére, pri porovnávaní jednotlivých krajín je ale tiež vhodné vyjadrenie emisií CO<sub>2</sub>eq na obyvateľa. Takto je následne možné porovnať, ako ku klimatickej zmene prispievajú rôzne veľké štáty.
   content:
   - emisie-sr
   - emisie-svet
   - emisie-svet-na-osobu
   - emisie-svet-na-hdp
   - emisie-eu-poradie
   - emisie-dychanie
+  qa:
+  - q: "Aké sú celkové svetové emisie?"
+    a: |
+      V roku 2012 boli celosvetové emisie 46 miliárd ton CO<sub>2</sub>eq, **v roku 2022 dosahujú hodnoty viac ako 50 miliárd ton CO<sub>2</sub>eq**.
+  - q: "Skleníkové plyny, CO<sub>2</sub>, CO<sub>2</sub>eq. Čo to vlastne znamená a aký je medzi týmito pojmami rozdiel?"
+    a: |
+      Poznáme viacero  <glossary id="antropogennesklenikoveplyny">skleníkových plynov</glossary>, **najvýznamnejší z nich je oxid uhličitý – CO<sub>2</sub>**. Jednotka *tona CO<sub>2</sub>* teda udáva výhradne množstvo oxidu uhličitého. V porovnaní s tým jednotka *tona CO<sub>2</sub>eq* (CO<sub>2</sub> ekvivalent) vyjadruje súhrnné množstvo skleníkových plynov prepočítaných na ekvivalentné množstvo CO<sub>2</sub>.
+  - q: "Ako sa emisie rôznych skleníkových plynov sčítajú dohromady na CO<sub>2</sub>eq?"
+    a: |
+      **Jednotlivé skleníkové plyny sa prepočítavajú na tzv. CO<sub>2</sub>eq**, (CO<sub>2</sub> ekvivalent), čo je množstvo oxidu uhličitého, ktoré by malo rovnaký príspevok ku skleníkovému javu atmosféry ako množstvo týchto ostatných plynov. Vzhľadom na rôzne polčasy rozkladu jednotlivých plynov v atmosfére sa tento príspevok uvažuje za určitú štandardizovanú dobu (obvykle 100 rokov) a vypočíta sa pomocou tzv. GWP (*Global Warming Potential*) koeficientov. Zatiaľ čo CO<sub>2</sub> ako referenčnému plynu prináleží koeficent 1, pre metán je to 28 pre horizont 100 rokov. Inými slovami, metán je 28× silnejší skleníkový plyn ako CO<sub>2</sub>.
+
+- id:          "slovensko"
+  title:       "Emisie skleníkových plynov Slovenska"
+  title-short: "Emisie v SR"
+  lead: |
+    V porovnaní s celosvetovými emisiami sa môžu zdať emisie Slovenska zanedbateľné – v roku 2018 vypustilo Slovensko [42,3 miliónov ton CO<sub>2</sub>eq](https://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=env_air_gge&lang=en). Keď ale emisie vyjadríme v prepočte na jedného obyvateľa, aby sme mohli slovenské emisie porovnať s inými štátmi, v roku 2015 priemerný obyvateľ Slovenska vyprodukoval 7,5 tony CO<sub>2</sub>eq, v roku 2018 to bolo už 7,8 tony (priemerné svetové emisie na osobu boli v týchto rokoch 6,5 tony, resp. 6,8 tony). **Priemerný obyvateľ Slovenska teda vyprodukuje ročne zhruba o 15 % viac emisií, ako je svetový priemer**.
```

Comment:
Tady to vypadá blbě, když se říká 42,3 Mt a současně grafika hned pod tím ukazuje 43,5 Mt. Opravil bych na to číslo v infografice. Hm?

Reply:
Hlasuju pro skrytí celé této sekce, dokud tu nebude víc infografik. Takto to vypadá dost zbytečně. Ten text kolem je hezký, ale až bude víc infografik.

Reply:
nefunguje mi preview, ale predpokladám, že myslíš SR emisný koláč, kde je také číslo... to pravdepodobne vzniklo spätnou úpravou dát v Eurostate, iný zmysel mi to nedáva (si pamätám, že sme si o tomto ešte na Slacku písali kedysi, keď mi nevychádzali počty). ach ach, tak to je ešte na pomerne major fix (nie je náhodou v pláne aktualizácia na novší rok?)

Reply:
> 

súhlasím, pozerala som diff a vyzerá to divne s jednou infografikou

Reply:
> Hlasuju pro skrytí celé této sekce, dokud tu nebude víc infografik. Takto to vypadá dost zbytečně. Ten text kolem je hezký, ale až bude víc infografik.

súhlasím, pozerala som diff a vyzerá to divne s jednou infografikou

Reply:
Přidal jsem tam TODO na sjednocení čísel, až se tato podsekce bude dávat ven. Teď to tak můžeme nechat.